devicon.json

```diff
@@ -5020,8 +5539,22 @@
         "color": "#00a1e0",
         "aliases": []
     },
+    {
+        "name": "rstudio",
+        "altnames": [],
+        "tags": ["editor", "package", "statistics"],
+        "versions": {
+          "svg": ["original", "plain"],
+          "font": ["plain"]
+        },
+        "color": "#75aadb",
+        "aliases": []
+    },
```

Comment:
Duplicate entry for `rstudio`

Reply:
6e9a895871f9c489551e347f4b9c05ee0830f8ae